Basic/DistributedAuthoritySocialHub/Assets/Scripts/Services/Vivox3DPositioning.cs

```diff
@@ -9,8 +9,15 @@ class Vivox3DPositioning : NetworkBehaviour
         bool m_Initialized;
         float m_NextPosUpdate;
 
-        void Start()
+        public override void OnNetworkSpawn()
         {
+            if (!HasAuthority)
+            {
+                base.OnNetworkSpawn();
```

Comment:
I believe every instance needs to invoke the base's method, or else we run into syncing issues.

Reply:
And if we have the unsubscriptions in OnDestroy, do we need unsubscriptions here?

Reply:
Good catch